.github/workflows/ci.yml

```diff
@@ -39,3 +39,23 @@ jobs:
 #        uses: github/super-linter@v4
 #        env:
 #          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  test:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: 11
+          distribution: "zulu"
+      - name: Remove Chrome
+        run: sudo apt purge google-chrome-stable
+      - name: Remove default Chromium
+        run: sudo apt purge chromium-browser
+      - name: Install a new Chromium
+        run: sudo apt install -y chromium-browser
+      - name: Test with Maven
+        run: mvn -Dtest=TestBrowser test
```

Comment:
Maybe better run all tests not only this specific one?

Reply:
I have to specify that one, othewise it will run the other runners that contain Android and iOS tests.